src/helpers/questionTypes.js

```diff
@@ -0,0 +1,20 @@
+export default {
+	'application/json;type=matching' : {
+		description: 'Simple Matching Question',
+		newItemSchema: {
+			name: 'Unnamed Matching Item',
+			instructions: '',
+			fromChoices: [],
+			toChoices: [],
+			answerConnections: []
+		}
```

Comment:
maybe add: components: { player: ..., customizer: ... } here instead of other file